package.json

```diff
@@ -98,6 +98,7 @@
 		"lodash.isfunction": "^3.0.9",
 		"lodash.memoize": "^4.1.2",
 		"lodash.reject": "^4.6.0",
+		"memoize-one": "^6.0.0",
```

Comment:
do we need some approval process to use a new lib?

Reply:
not sure, @interactivellama?

Reply:
We have license and security checking from Snyk in our CI, so it should conform to Snyk's policy https://app.snyk.io/org/salesforce-oss/pr-checks/ab6331d2-5eb8-4973-8bf5-aab69313e91f